packages/grid/stories/grid.doc.stories.mdx

```diff
@@ -148,24 +169,60 @@ This grid also shows an example of a custom cell displaying a basic chart. Switc
 
 ### Editable cells
 
-Grid allows you to make cell values editable in place.
-Use the `CellEditor` component as a child of `GridColumn` to make the column editable.
-`CellEditor` is a non-rendered component.
-`CellEditor` is expected to have one child representing the specific editor component to use for the column, e.g. `TextCellEditor`, `DropdownCellEditor` or `NumericCellEditor`.
+The grid allows you to make cell values editable in place.
+
+Use the `CellEditor` (a non-rendered) component as a child of `GridColumn` to make the column editable.
```

Comment:
Do we explain what a "non-rendered" component is anywhere in the docs? Might be useful to explain the concept, and it might prevent our users doing funky stuff with the children of the Grid in hopes of achieving different layouts.

Reply:
docs updated